docs/src/kernel/linux-compatibility/limitations-on-system-calls/file-systems-and-mount-control.md

```diff
@@ -7,3 +7,73 @@ fdatasync, sync, syncfs, sync_file_range, open_tree, move_mount, fsopen,
 fsconfig, fsmount, and fspick
 under this category.
 -->
+
+## Mount and Unmount File Systems
+
+### `mount`
+
+Supported functionality in SCML:
+
+```c
+// Move the existing mount point atomically
```

Comment:
This is misleading: it looks that Asterinas does not support creating a new mount at all. This is the most basic mount operation. Revise the doc this way.

```c
// Create a new mount
mount(
    source, target, filesystemtype,
    mountflags = 0,
    data
);

// Move the existing mount point
mount(
    source, target, filesystemtype,
    mountflags = MS_MOVE,
    data
);
```